charts/camunda-platform-8.8/templates/orchestration/migration-identity-configmap.yaml

```
@@ -33,7 +33,7 @@ data:
             {{- end }}
           management-identity:
             audience: {{ include "identity.authAudience" . | quote }}
-            base-url: {{ tpl .Values.global.identity.auth.identity.redirectUrl $ | quote }}
+            base-url: http://{{ include "identity.fullname" . }}:80
```

```suggestion
            base-url: http://{{ include "identity.fullname" . }}:80{{ .Values.identity.contextPath }}
```

not sure it needs an extra quoting but generally worked when we tried it out earlier without the quoting

---

I think it works because this is converted from spring to string